scilpy/tractanalysis/scoring.py

```diff
@@ -25,10 +25,11 @@
     - OL : percentage of ground truth voxels containing VS streamline(s).
     - OR/ORn: percentage of voxels containing VS streamline(s) when it
```

Comment:
Change this line to `OR_vs/OR_gt: percentage of voxels containing VS streamline(s) when it` ?